EMIP/1-1000/EMIP00099.xml

```diff
@@ -42,8 +42,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
 
                      <msItem xml:id="ms_i1.1">
                         <locus from="2r" to="158r" facs="003"/>
-                        <title type="complete" ref="LIT3985Mazmura"/>
+                        <title type="complete" ref="LIT2000Mazmur"/>
                         <note><ref type="work" corresp="LIT2000Mazmur#Ps118Spiritual"/></note>
+                        <note>Each psalm includes at its end the text of the Psalter of the Virgin after</note>
```

Comment:
if this note is correct then LIT3985Mazmura is the correct CAe ID - Mazmura dengel